docs/user-guide/configuring.md

```diff
@@ -349,12 +349,11 @@ In these configuration files, the rule `plugin1/rule1` comes from the plugin nam
 
 ## Disabling Rules with Inline Comments
 
-To temporarily disable rule warnings in your file use the following format:
+To temporarily disable rule warnings in your file, use block comments with the following format:
 
 ```js
 /* eslint-disable */
 
-// Disables all rules between comments
```

Comment:
After looking at this again, having these explanation comments is REALLY confusing when we're telling users to look at how to format comments to disable rules inline. I think it's clearer with them gone.